tests/ast.cpp

```diff
@@ -51,11 +51,15 @@ TEST(ast, probe_name_uprobe)
   AttachPointList attach_points3 = { &ap1, &ap2, &ap3 };
   Probe uprobe3(&attach_points3, nullptr, nullptr);
   EXPECT_EQ(uprobe3.name(), "uprobe:/bin/sh:readline,uprobe:/bin/sh:somefunc,uprobe:/bin/sh:1000");
-
```

Comment:
Could you leave this spacing line in please? I feel it makes the tests a lot more readable

Reply:
Oops, sloppy.